### PR TITLE
Feature/allow var tmp

### PR DESCRIPTION
**Pull request by dmjacobsen**
for issue #262

### CODE REVIEW

File: src/VolumeMap.c [hunk 1]
```diff
@@ -585,6 +589,7 @@ int _validateVolumeMap(
         const char *from,
         const char *to,
         VolumeMapFlag *flags,
+	const char **toExactAllowed,
```
Comment (agaur-lbl) [inline src/VolumeMap.c]: space ?

**Comment by agaur-lbl** [comment]
Overall looks good, i would like paths to be configurable. Special paths can be given special consideration.

File: src/VolumeMap.c (hunk 1 above)
Comment (dmjacobsen) [reply src/VolumeMap.c]: doh! i'll fix that

**Comment by dmjacobsen** [comment]
the spacing is corrected.  It looks like travisci fell apart on the previous push owing to a transient failure at travis.  hopefully this push will build.

**Comment by dmjacobsen** [comment]
re: making paths configurable, i believe that is in-general possible.  /var is a little touchy owing to its security issues.  I don't think there is any harm in /var/tmp, but other paths within var could be more problematic.  We can re-address the overall configurability in a branch focused on that.

**Pull request merged**